config/initializers/db2fog.rb

```diff
@@ -0,0 +1,7 @@
+# See: https://github.com/yob/db2fog
+DB2Fog.config = {
+    :aws_access_key_id     => Spree::Config[:s3_access_key],
```

Comment:
so if these keys are empty does it fail loudly? or it simply skips?

Reply:
I was considering wrapping it in an `if` statement and checking `Spree::Config[:s3_bucket]` is set... what's the best way to do this?

Reply:
We have a cron job for db2fog in `scedule.rb` here: https://github.com/openfoodfoundation/openfoodnetwork/blob/07888b355a2888e118d6d59ce968ab88125ebadc/config/schedule.rb#L28

We don't configure this anywhere with ofn-install as far as I can tell, so this job always runs.

Reply:
Looking at how the previous deprecated initializer was being loaded, and how things are set on servers that don't use S3 backups, I'd say that it quietly ignores or skips if the initializer settings are empty. 

I don't think this will be any different to the way it's being loaded currently, which is this: https://github.com/openfoodfoundation/openfoodnetwork/blob/master/config/initializers/0_depricated_db2fog.rb

Is that right?

Reply:
The current initializer is prexifed with a zero in the filename so it gets loaded first, then any other initializer that gets added after (by ofn-install) that does the same configuration will overwrite it. That's the current setup. So we're keeping the exact same logic and initialization, but removing all the gymnastics performed in ofn-install to add a second initializer that overrides the first.

Reply:
> We don't configure this anywhere with ofn-install as far as I can tell, so this job always runs.

And it always fails.

I think the simplest is to add the conditional on schedule.rb so that we don't schedule useless jobs, but IMO it is a separate issue that can be tackled in a separate PR.